### PR TITLE
[docs] Resolved Edit Button

### PR DESCRIPTION
**Pull request by mosespace**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] I have included in this PR's description the statement `closes #xx` or `fixes #xx` where #xx is the number of the issue being addressed.

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
  This pull request fixes a bug / feature in the **`.yml`** file

* **What is the current behavior?** (You can also link to an open issue here)
  It was noticed that the edit link on the documentation references master branch of which we don't have a branch named master in issue! #140 

* **What is the new behavior (if this is a feature change)?**
  Now in this new feature, the edit button is removed hidden/removed!

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
  This pull request does not introduce any breaking change!

* **Other information**:
  This pr closes issue #140 
  This pr closes issue #138 
  This pr closes issue #137 
  This pr closes issue #143

### CODE REVIEW

**Comment by HassanBahati** [comment]
I think we should be fixing the branch in reference than getting rid of the feature as a whole

File: mkdocs.yml [hunk 1]
```diff
@@ -90,6 +92,9 @@ markdown_extensions:
   # Plugins: This is a plugin that enables the search engine
   #in the markdown file.
 
+  # this option removes the edit buttton
+edit_uri: ""
```
Comment (HassanBahati) [inline mkdocs.yml:96]: we would rather add this `edit_uri: blob/main/docs/` than eliminate it

Comment (mosespace) [reply mkdocs.yml:96]: okay, done this

**Comment by mosespace** [comment]
@HassanBahati , any review on this?

**Pull request merged**